cuegui/cuegui/MainWindow.py

```diff
@@ -120,6 +120,15 @@ def showStatusBarMessage(self, message, delay=5000):
     def displayAbout(self):
         """Displays about text."""
         msg = self.app_name + "\n\nA opencue tool\n\n"
+        msg += "CueGUI:\n%s\n\n" % cuegui.Constants.VERSION
+
+        # You can add more details like beta/stable indication here
+        beta_version = os.getenv('OPENCUE_BETA', '0')
+        if beta_version == '1':
```

Comment:
This can be simplified into a single expression. The value of `OPENCUE_BETA` doesn't need to be `'1'`, it simply needs to be defined.
```suggestion
        if os.getenv('OPENCUE_BETA'):
```